packages/test-e2e/test/sdk/cjs/types/Uniswap.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Hmmm why this is empty?

Reply:
Well, since we emit _Uniswap.ts_ from TypeChain, we transpile it in eth-sdk and it doesn't include any types, we get an empty module.